### ISSUE TITLE
ShapeWorks 6.1 Testing: Version of glibc

### ISSUE THREAD

**Issue by patkins** [SCIInstitute/ShapeWorks#1279]
During  #1273  testing of the release candidate for ShapeWorks 6.1, I came upon the following issue: Using the linux artifact on OpenSUSE Leap 15.2, it seems glibc 2.27 is required, but I did not receive any errors during the install. 

```
(shapeworks) atkins@localhost:~/programs/ShapeWorks-dev-3730-b593ce7f-linux/bin> ./ShapeWorksStudio 
./ShapeWorksStudio: /lib64/libm.so.6: version `GLIBC_2.27' not found (required by ./ShapeWorksStudio)
./ShapeWorksStudio: /lib64/libm.so.6: version `GLIBC_2.27' not found (required by /home/atkins/programs/ShapeWorks-dev-3730-b593ce7f-linux/bin/../lib/libITKTransformFactory-5.0.so.1)
./ShapeWorksStudio: /lib64/libm.so.6: version `GLIBC_2.27' not found (required by /home/atkins/programs/ShapeWorks-dev-3730-b593ce7f-linux/bin/../lib/libvtkCommonDataModel-8.2.so.1)
./ShapeWorksStudio: /lib64/libm.so.6: version `GLIBC_2.27' not found (required by /home/atkins/programs/ShapeWorks-dev-3730-b593ce7f-linux/bin/../lib/libitkhdf5.so.1)
./ShapeWorksStudio: /lib64/libm.so.6: version `GLIBC_2.27' not found (required by /home/atkins/programs/ShapeWorks-dev-3730-b593ce7f-linux/bin/../lib/libitkgdcmMSFF-5.0.so.1)
./ShapeWorksStudio: /lib64/libm.so.6: version `GLIBC_2.27' not found (required by /home/atkins/programs/ShapeWorks-dev-3730-b593ce7f-linux/bin/../lib/libv3p_netlib.so)

(shapeworks) atkins@localhost:~/programs/ShapeWorks-dev-3730-b593ce7f-linux/bin> /lib64/libc.so.6 --version
GNU C Library (GNU libc) stable release version 2.26 (git 1c9a5c270d8b), by Roland McGrath et al.
Copyright (C) 2017 Free Software Foundation, Inc.
This is free software; see the source for copying conditions.
There is NO warranty; not even for MERCHANTABILITY or FITNESS FOR A
PARTICULAR PURPOSE.
Configured for x86_64-suse-linux.
Compiled by GNU CC version 7.5.0.
Available extensions:
        crypt add-on version 2.1 by Michael Glad and others
        GNU Libidn by Simon Josefsson
        Native POSIX Threads Library by Ulrich Drepper et al
        BIND-8.2.3-T5B
libc ABIs: UNIQUE IFUNC
For bug reporting instructions, please see:
<http://bugs.opensuse.org>.

```

**Comment by akenmorris** [comment]
Hi @patkins , can you try this build and let me know if the executables work for you?  I believe they should:

www.sci.utah.edu/~amorris/ShapeWorks-v6.0.0-875-g2bbc1308-linux.zip

**Comment by patkins** [comment]
@akenmorris that version seems to work on OpenSUSE15.2!

**Comment by akenmorris** [comment]
Fix in #1289
